src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -19,7 +19,7 @@ namespace Dynamo.Nodes
 {
     [NodeName("String")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("Creates a string.")]
+    [NodeDescription("StringInputNodeDescription", typeof(Properties.Resources))]
```

Comment:
Hi @holyjewsus and @Benglin , I have fixed this. please take a look. 
https://github.com/DynamoDS/Dynamo/pull/3634